app/src/pages/admin/serverSettingsPage/serverSettingsTabs/linksAndBrandingTab/addLinkForm/addLinkForm.jsx

```diff
@@ -91,7 +91,7 @@ const AddLink = ({ onClose, handleSubmit, customLinks }) => {
           fieldWrapperClassName={cx('field-wrapper')}
         >
           <FieldErrorHint>
-            <Input />
+            <Input maxLength={'1024'} />
```

Comment:
Maybe it will be better to not cut user input by length?
In this case the user may accidentally save the broken url.
Pls discuss with QA team ability to replace it with max length validation instead.